org.jacoco.core/src/org/jacoco/core/internal/flow/MethodProbesVisitor.java

```diff
@@ -147,4 +148,8 @@ public void visitLookupSwitchInsnWithProbes(final Label dflt,
 			final int[] keys, final Label[] labels, final IFrame frame) {
 	}
 
+	public void accept(final MethodNode methodNode,
```

Comment:
> Nevertheless we should clearly document the method, especially the point in time when it is called.

Not sure I understand this point - do you want to document method or its call site or both? And in any case - do you have idea about wording?